lms/djangoapps/dashboard/tests/test_sysadmin.py

```diff
@@ -120,6 +122,126 @@ def _mkdir(self, path):
         self.addCleanup(shutil.rmtree, path)
 
 
+@unittest.skipUnless(
+    settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD'),
+    "ENABLE_SYSADMIN_DASHBOARD not set",
+)
+class TestSysadminEditCourseTabs(SysadminBaseTestCase):
+    """Tests for sysadmin edit_course_tabs tools"""
+
+    def setUp(self):
+        super(TestSysadminEditCourseTabs, self).setUp()
+        self.course = CourseFactory.create(org="test", course="course1", display_name="run1")
```

Comment:
Looking around at other calls to `CourseFactory.create`, it seems like the kwargs may be optional; can we try omitting them here? Presumably, the factory uses sane defaults.